lib/Doctrine/ODM/MongoDB/Mapping/LegacyReflectionFields.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ODM\MongoDB\Mapping;
+
+use ArrayAccess;
+use Countable;
+use Doctrine\ORM\Mapping\ReflectionReadonlyProperty;
```

Comment:
This class is missing. There's a `ReadonlyAccessor`, but in my project with readonly properties I'm getting an error because of this missing class.

```suggestion
use Doctrine\ODM\MongoDB\Mapping\ReflectionReadonlyProperty;
```

Reply:
Good catch, I imported the missing class from ORM.